optimize/backend/src/test/java/io/camunda/optimize/service/ProcessOverviewServiceTest.java

```diff
@@ -0,0 +1,194 @@
+
+    // then
+    assertThat(result).hasSize(1);
+    final ProcessOverviewResponseDto response = result.get(0);
+    assertThat(response.getProcessDefinitionKey()).isEqualTo(processKey);
+    assertThat(response.getProcessDefinitionName()).isEqualTo(processName);
+    assertThat(response.getOwner().getId()).isEqualTo("owner123");
+    assertThat(response.getOwner().getName()).isEqualTo("Owner Name");
+  }
+
+  @Test
+  void shouldUpdateProcessWhenAuthorizedAndValid() {
+    // given
+    final String userId = "user123";
+    final String processKey = "processKey1";
+    final String newOwnerId = "owner456";
+
+    final ProcessUpdateDto updateDto =
+        new ProcessUpdateDto(newOwnerId, new ProcessDigestRequestDto(true));
+
+    final List<String> tenants = new ArrayList<>();
+    tenants.add("1");
+
+    when(definitionService.getProcessDefinitionWithTenants(processKey))
+        .thenReturn(
+            Optional.of(
+                new DefinitionWithTenantIdsDto(
+                    processKey, "Process Name", PROCESS, tenants, Set.of())));
+    when(definitionAuthorizationService.isAuthorizedToAccessDefinition(any(), any(), any(), any()))
+        .thenReturn(true);
+
+    when(identityService.getUserById(newOwnerId)).thenReturn(Optional.of(new UserDto(newOwnerId)));
+    when(identityService.isUserAuthorizedToAccessIdentity(
+            userId, new IdentityDto(newOwnerId, IdentityType.USER)))
+        .thenReturn(true);
+
+    // when
+    processOverviewService.updateProcess(userId, processKey, updateDto);
+
+    // then
+    verify(processOverviewWriter).updateProcessConfiguration(processKey, updateDto);
+    verify(digestService).handleProcessUpdate(processKey, updateDto);
+  }
+
+  @Test
+  void shouldThrowExceptionWhenUnauthorizedForProcessUpdate() {
+    // given
+    final String userId = "user123";
+    final String processKey = "processKey1";
+
+    final ProcessUpdateDto updateDto =
+        new ProcessUpdateDto("owner456", new ProcessDigestRequestDto(true));
+
+    final List<String> tenants = new ArrayList<>();
+    tenants.add("1");
+
+    when(definitionService.getProcessDefinitionWithTenants(processKey))
+        .thenReturn(
+            Optional.of(
+                new DefinitionWithTenantIdsDto(
+                    processKey, "Process Name", PROCESS, tenants, Set.of())));
+    when(definitionAuthorizationService.isAuthorizedToAccessDefinition(
+            userId, PROCESS, processKey, List.of("tenant1")))
+        .thenReturn(false);
+
+    // when
+    final Throwable thrown =
+        catchThrowable(() -> processOverviewService.updateProcess(userId, processKey, updateDto));
+
+    // then
+    assertThat(thrown).isInstanceOf(ForbiddenException.class);
+  }
+
+  @Test
+  void shouldUpdateOwnerIfDefinitionIsImported() {
+    // given
+    final String userId = "user123";
+    final String processKey = "processKey1";
+    final String ownerId = "owner456";
```

Comment:
maybe we can make these constants rather than initializing them for all of the tests?

Reply:
fixed